bin/ungit

```diff
@@ -58,11 +58,11 @@ const launch = () => {
   );
 
   child.on('exit', (res) => {
-    console.log('Stopped keeping ungit alive');
+    console.log('Stopped! keeping ungit alive');
   });
 
   const startupListener = (data) => {
-    if (data.toString().indexOf('## Ungit started ##') >= 0) {
```

Comment:
Ungit's clicktest capture this console output to detect the service start and this is the reason why all tests are failing.

Reply:
@jung-kim how to change this behavior?

Reply:
Before that, why do you want to change this behavior?

Reply:
the phrase `## Ungit started ##` with `##` seems ugly for me !

Reply:
it seems that what is being suggested here is about the appearance of string, I'm not sure that is a strong enough of a reason to change test behaviors.

Reply:
As you want! I think it's not that difficult to change test behavior! 
Have a nice day.